book/cli/reth/node.md

```diff
@@ -367,6 +367,9 @@ RPC:
 
           [default: 25]
 
+      --flashbots.blacklist <PATH>
+          Path to file containing blacklisted addresses, json-encoded list of strings. Block validation API will reject blocks containing transactions from these addresses
```

Comment:
let's rename this to --builder.disallow

and replace blacklist terminology with disallow